### DISCUSSION TITLE
Debugging webview extensions content

### DISCUSSION THREAD

**Discussion by gocal** [microsoft/vscode-discussions#1159] [Extension Development QnA] [answered]
Is it possible to debug the content of the webview extension? I am using the [weather-weview](https://github.com/microsoft/vscode-webview-ui-toolkit-samples/tree/main/default/weather-webview) example as a reference and I would like to place a breakpoint directly in the code that is hosted with [webview](https://github.com/microsoft/vscode-webview-ui-toolkit-samples/blob/main/default/weather-webview/src/webview/main.ts).

**Comment by tamuratak** [comment, marked as answer]
You can do that by setting `"debugWebviews": true` in `.vscode/launch.json`. 

- https://github.com/microsoft/vscode-js-debug/blob/main/OPTIONS.md#debugwebviews

**Comment by abrilurena** [comment]
Hi, @gocal! do you still need help with this?
